go.sum

```diff
@@ -1,20 +1,16 @@
-cloud.google.com/go v0.26.0 h1:e0WKqKTd5BnrG8aKH3J3h+QvEIQtSUcf2n5UZ5ZgLtQ=
 cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
Regarding go.sum it can make prs unclean. Maybe we could ignore the file

Reply:
No. You should set your `-mod` to `vendor` or `readonly` manually or in `GOFLAGS` and not checkin go.sum changes unless you are updating  dependencies.

https://github.com/golang/go/wiki/Modules

Reply:
ok i will update it

Reply:
thx

Reply:
done